### PR TITLE
Adding 2 test files

### PR DESCRIPTION
**Pull request by hanyelgaml-roche**

### CODE REVIEW

File: src/efs_parser/BaseFile.py [hunk 1]
```diff
@@ -219,12 +219,19 @@ def bin_rows(self, data, chr, start, end, columns=None, metadata=None, bins = 40
             temps = data[(data.index.left <= index.right) & (data.index.right > index.left)]
             if len(temps) > 0:
                 for col in columns:
-                    row[col] = float(np.mean(temps[col].values))
+                    #row[col] = float(np.mean(temps[col].values))
+                    row[col] = np.mean(self.change_to_float(temps[col].values))
```
Comment (jkanche) [inline src/efs_parser/BaseFile.py]: @hanyelgaml-roche 

use the pandas astype method instead - https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.astype.html#pandas-dataframe-astype

**Review by jkanche — CHANGES_REQUESTED** [review]

File: src/efs_parser/BaseFile.py [hunk 2]
```diff
@@ -220,18 +220,14 @@ def bin_rows(self, data, chr, start, end, columns=None, metadata=None, bins = 40
             if len(temps) > 0:
                 for col in columns:
                     #row[col] = float(np.mean(temps[col].values))
-                    row[col] = np.mean(self.change_to_float(temps[col].values))
+                    #row[col] = np.mean(self.change_to_float(temps[col].values))
+                    values_to_fix = [float(i) for i in temps[col].values]
```
Comment (jkanche) [inline src/efs_parser/BaseFile.py]: this doesn't work ? 

```
temps[col].apply(float)
```

or this ?

```
temps[col].astype(float64)
```

Comment (hanyelgaml-roche) [reply src/efs_parser/BaseFile.py]: These 2 work:
temps[col].apply(float)
temps[col].astype(float)
However, this does work:
temps[col].astype(float64)

Thank you for the hint. I will update the code accordingly

Comment (jkanche) [reply src/efs_parser/BaseFile.py]: ok, Iets use the pandas inbuilt method `astype`

```temps[col].astype(float)```

Comment (hanyelgaml-roche) [reply src/efs_parser/BaseFile.py]: I have committed the change

**Pull request merged**